tasks/parse-pdr/index.js

```diff
@@ -36,6 +37,17 @@ function parsePdr(event) {
   return parse.ingest()
     .then((payload) => {
       if (parse.connected) parse.end();
+      // opportunity to filter the granules of interest based on regex in granuleIdExtraction
+      if (config && config.collection && config.collection.granuleIdExtraction) {
```

Comment:
This is referencing config.collection.granuleIdExtraction, but there is no collection in the config schema:

https://github.com/cumulus-nasa/cumulus/blob/master/tasks/parse-pdr/schemas/config.json

I would recommend adding an optional "granuleIdFilter" property to the config schema and use that instead of config.collection.granuleIdExtraction.

Reply:
Will do.

Reply:
but collection would still be optional...

Reply:
The parse-pdr task does not currently take a collection as part of its config.  If this change is made, it should _still_ not take a collection as part of its config.  It should take a property called "granuleIdFilter".